Pod/Classes/Logo/UIView+Logo.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  UIView+Logo.swift
+//  Pods
+//
+//  Created by Derek Ostrander on 7/18/16.
+//
+//
+
+import UIKit
+
+public extension UIView {
+    private class RZBundle {} // to grab the image url
+    public static func rz_logo() -> UIImageView {
+        guard let url = NSBundle(forClass: RZBundle.self).URLForResource("logo-built-by-RZ", withExtension: "png"),
```

Comment:
#PP one of the few times I'll say this, is there a good reason we're not using a .PDF instead of .png files? That means that if another image scale comes out we wouldn't have to update the library